yt/frontends/gamer/fields.py

```diff
@@ -59,42 +69,110 @@ def setup_fluid_fields(self):
         unit_system = self.ds.unit_system
         unit_system.registry = self.ds.unit_registry  # TODO: Why do I need this?!
 
+        temp_conv = pc.kb / (self.ds.mu * pc.mh)
```

Comment:
How about setting `temp_conv = 1 / mu` for dimensionless simulations to be more consistent with GAMER (e.g., see [here](https://github.com/gamer-project/gamer/blob/main/src/EoS/TaubMathews/CPU_EoS_TaubMathews.cpp#L56-L57))? So something like
```
        if self.ds.unit:
            temp_conv = pc.kb / (self.ds.mu * pc.mh)
        else:
            temp_conv = self.ds.arr(1.0, "code_velocity**2/code_temperature") / self.ds.mu
```
where `self.ds.unit` is set in `data_structures.py` (see the comment therein). Also check the following comments on other physical constants.

Reply:
Done.